static/app/utils/profiling/profile/utils.tsx

```diff
@@ -8,6 +8,24 @@ import {CallTreeNode} from '../callTreeNode';
 
 type FrameIndex = Record<string | number, Frame>;
 
+export function createSentrySampleProfileFrameIndex(
+  frames: Profiling.SentrySampledProfile['profile']['frames']
+): FrameIndex {
+  const frameIndex: FrameIndex = {};
+
+  for (let i = 0; i < frames.length; i++) {
+    const frame = frames[i];
+
+    frameIndex[i] = new Frame({
+      key: i,
+      name: frame.name ?? 'unknown',
+      line: frame.line,
```

Comment:
lets add instruction_addr and possibly column here if it's supported cc @phacops

Reply:
The field will be named `colno`.